test/unit/spec/EnrollCall_spec.js

```diff
@@ -585,6 +585,38 @@ function (Q, Okta, OktaAuth, LoginUtil, Util, Form, Beacon, Expect, $sandbox,
           });
       });
 
+      itp('on WARNING_TIMEOUT, after dial/redail, shows warning message for Call after 30s idle', function () {  
```

Comment:
text should be just `shows warning message to click "Redial" after 30s`, cause there is no reference to WARNING_TIMEOUT in the tests. And actually that variable is only defined in PushForm

Reply:
fixed